internal/dcache/rpc/client/functions.go

```diff
@@ -307,6 +314,7 @@ func EndSync(ctx context.Context, targetNodeID string, req *models.EndSyncReques
 
 // cleanup closes all the RPC node client pools
 func Cleanup() error {
+	log.Debug("rpc_client::Cleanup: Closing all node client pools")
```

Comment:
this can be Info, as it's rare and it's good to know
same for L327

Reply:
Updated